apps/postybirb-ui/src/components/form/fields/form-field.type.ts

```diff
@@ -1,5 +1,6 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import { UseFormReturnType } from '@mantine/form';
+// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
```

Comment:
q: why are these now being added? new lint rule?

Reply:
ig this is already resolved, but still -> https://github.com/mvdicarlo/postybirb/pull/302#discussion_r1817770303